cinema/serializers.py

```diff
@@ -57,6 +68,7 @@ class MovieSessionListSerializer(MovieSessionSerializer):
         source="cinema_hall.capacity", read_only=True
     )
     tickets_available = serializers.IntegerField(read_only=True)
+    movie_image = serializers.CharField(source="movie.image", read_only=True)
```

Comment:
it should be ImageField, because with CharField link won't be displayed properly